libraries/AP_Camera/AP_Camera.h

```diff
@@ -118,6 +118,11 @@ class AP_Camera {
     // take multiple pictures, time_interval between two consecutive pictures is in miliseconds
     // total_num is number of pictures to be taken, -1 means capture forever
     void take_multiple_pictures(uint32_t time_interval_ms, int16_t total_num);
+    void take_multiple_pictures(uint8_t instance, uint32_t time_interval_ms, int16_t total_num);
```

Comment:
I think we should get in the habit of returning booleans so that caller knows if the command succeeded or not. In these cases it probably only fails if the instance is incorrect but in the future we could add other reasons, for example if the interval was below the camera's minimum interval speed it could fail.

Reply:
done for all three function which are introduced in this PR